abci/specification.rst

```diff
@@ -1,294 +0,0 @@
-ABCI Specification
-==================
-
-NOTE: this file has moved to `specification.md <./specification.md>`__. It is left to prevent link breakages for the forseable future. It can safely be deleted in a few months.
```

Comment:
this note applied to the abci repo. The current spec is in `docs/abci-spec.md` and links will be updated to point there from the old abci repo location. Nothing should link to this file since it's only a week or so in this location.